src/nft/hooks/useCollectionCount.ts

```diff
@@ -0,0 +1,20 @@
+import create from 'zustand'
+import { devtools, persist } from 'zustand/middleware'
+
+interface State {
+  collectionCount: number
+  setCollectionCount: (collectionCount: number) => void
+}
+
+export const useCollectionCount = create<State>()(
+  persist(
+    devtools(
+      (set) => ({
+        collectionCount: 0,
+        setCollectionCount: (collectionCount) => set({ collectionCount }),
```

Comment:
the collection count is fetched a level down from the parent where filter button is rendered.  This didn't use the case, but was changed.  I created state here to keep track of that value